modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractCppCodegen.java

```diff
@@ -128,6 +128,11 @@ public String escapeReservedWord(String name) {
     }
 
     @Override
+    public String toOperationId(String operationId) {
+        return sanitizeName(super.toOperationId(operationId));
```

Comment:
What about the case if the parameter is an empty string? If its not overwritten, it checks for that.

Reply:
This case is covered by the call to the default implementation (`super.toOperationId`), which throws if the string is empty.

Reply:
@fvarose what about using something similar to [RubyClientCodegen.java#L619-L635](https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/RubyClientCodegen.java#L619-L635)?

It just a matter of time we need to handle these edge cases.